src/apps/core/submodules/alerts/alerts.js

```diff
@@ -16,10 +16,47 @@ define(function(require) {
 		 * Trigger the alerts pulling process from API.
 		 */
 		alertsRender: function() {
-			var self = this;
+			var self = this,
+				initTemplate = function initTemplate(alerts) {
+					var alertGroups = alerts ? self.alertsFormatData({ data: alerts }) : {},
+						alertsCount = _.reduce(alertGroups, function(count, alertGroup) {
+							return count + alertGroup.length;
+						}, 0),
+						dataTemplate = {
+							alertsCount: alertsCount === 0 ? null : alertsCount > 9 ? '9+' : alertsCount.toString()
```

Comment:
Ternaries should not be nested, as it make its harder to understand the ongoing logic.

You could use two different props, one to determine if the alert count and one to contain the actual alert count.